fhir-persistence-schema/README.md

```diff
@@ -1,58 +1,37 @@
+This module is built into two different jar files. The default jar is included with the IBM FHIR Server web application and is used for bootstrapping Apache Derby databases (if configured). There is also an executable command line interface (cli) version of this jar that packages this module with all of its dependencies.
 
-This module is built into two different jar files. The executable command line interface (cli) version of this module is can also be downloaded directly from [JFrog Bintray](https://bintray.com/ibm-watson-health/ibm-fhir-server-releases/fhir-persistence-schema#files/com/ibm/fhir/fhir-persistence-schema) or is available from the Releases tab.
+The executable command line interface (cli) version of this module can be downloaded from the project [Releases tab](https://github.com/IBM/FHIR/releases).
 
-The cli jar is included in the FHIR Server distribution zip but 
+## Getting started
+### Creating the database
 
-## Execute the fhir-persistence-schema command line interface (CLI)
-
-1 - Execute the corresponding command, where VERSION is the version of the fhir-persistence-schema cli jar that you are using, and SCHEMA_COMMAND is the documented schema command you are using.
-
-*Linux/Mac*  
-
-```
-VERSION=4.0.0-SNAPSHOT
-java -jar ./fhir-persistence-schema-${VERSION}-cli.jar SCHEMA_COMMAND
-```
-
-*Windows*
-
-```
-set VERSION=4.0.0-SNAPSHOT
-java -jar ./fhir-persistence-schema-%VERSION%-cli.jar SCHEMA_COMMAND
-```
+To create the Db2 database and database user, use the following commands:
 
-## Creating the database
-
-To create the database and database user, please use the following command.
-
-``` shell 
-useradd fhiruser
+``` shell
+useradd fhirserver
 su - db2inst1 -c "db2 CREATE DB FHIRDB using codeset UTF-8 territory us PAGESIZE 32768"
-su - db2inst1 -c "db2 \"connect to fhirdb\" && db2 \"grant connect on database TO USER fhiruser\""
+su - db2inst1 -c "db2 \"connect to fhirdb\" && db2 \"grant connect on database TO USER fhirserver\""
 ```
 
-## Printing the schema
+### Printing the schema
 
 To print the schema DDL for review, execute `com.ibm.fhir.schema.app.SchemaPrinter`:
 
-``` shell 
+``` shell
 java -cp ./fhir-persistence-schema-${VERSION}-cli.jar com.ibm.fhir.schema.app.SchemaPrinter [--to-file]
 ```
 
-Without to-file, the output is the current System.out; else it's schema.sql, grants.sql, and stored-procedures.sql of the current directory.
+With `--to-file` it outputs to `./schema.sql`, `./grants.sql`, and `./stored-procedures.sql`; otherwise to System.out.
 
-
-## Deploy and manage the schema with `com.ibm.fhir.schema.app.Main` 
+## Execute the fhir-persistence-schema command line interface (CLI)
 
 To deploy and manage the schema on a target database, the `fhir-persistence-schema` project includes a Main class that can parallelize the schema updates.
 
-``` shell 
+``` shell
 java -jar ./fhir-persistence-schema-${VERSION}-cli.jar [OPTIONS]
 ```
```

Comment:
```suggestion
```

On Windows use `%VERSION%`
```

Reply:
may also need to specify where the VERSION comes from

Reply:
I also thought to just make it `*` like we just did in our script...then its platform-neutral, right?

Reply:
added a note to replace version with the version